site/docs/install-windows.md

```diff
@@ -35,11 +37,11 @@ Alternatively you can:
 
 *   [Download Bazelisk](https://github.com/bazelbuild/bazelisk) instead of Bazel. Bazelisk is a
     Bazel launcher that ensures you always use the latest Bazel release.
-*   [Install Bazel from Chocolatey](#install-using-chocolatey) (see below)
-*   [Install Bazel from Scoop](#install-using-scoop) (see below)
+*   [Install Bazel from Chocolatey](#using-chocolatey)
+*   [Install Bazel from Scoop](#using-scoop)
 *   [Build Bazel from source](install-compile-source.html)
 
-## 4. Optional: configure output directories
+### Step 4: Optionally configure output directories
```

Comment:
I think optional: install   or   (optional) install    is  better than optionally install.
You are designating an optional step, not saying that you should install it in an optional fashon.

Reply:
Done.